letta/client/streaming.py

```diff
@@ -59,8 +59,8 @@ def _sse_post(url: str, data: dict, headers: dict) -> Generator[LettaStreamingRe
                             yield ToolCallMessage(**chunk_data)
                         elif "tool_return" in chunk_data:
                             yield ToolReturnMessage(**chunk_data)
-                        elif "usage" in chunk_data:
-                            yield LettaUsageStatistics(**chunk_data["usage"])
+                        elif "step_count" in chunk_data:
```

Comment:
Slightly weird that we're using `step_count` here but that's fine

Reply:
yeah it was an abritrary choice but if we move to nesting in the future we can change this back